packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -98,4 +117,48 @@ contract SuperchainWETH is WETH98, IERC7802, ISemver {
         return _interfaceId == type(IERC7802).interfaceId || _interfaceId == type(IERC20).interfaceId
             || _interfaceId == type(IERC165).interfaceId;
     }
+
+    /// @notice Sends ETH to some target address on another chain.
+    /// @param _to       Address to send tokens to.
+    /// @param _chainId  Chain ID of the destination chain.
+    function sendETH(address _to, uint256 _chainId) external payable returns (bytes32 msgHash_) {
+        if (_to == address(0)) revert ZeroAddress();
+
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
+            revert NotCustomGasToken();
+        }
+
+        IETHLiquidity(Predeploys.ETH_LIQUIDITY).burn{ value: msg.value }();
+
+        msgHash_ = IL2ToL2CrossDomainMessenger(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER).sendMessage({
```

Comment:
small nit: make `messenger` an internal variable defined at the top of the file so you don't have to inline like this

Reply:
setting `messenger` to an internal variable does not seem to play well with solidity `0.8.15` (which is required by WETH98):
![Screenshot 2024-11-20 at 4 53 22 PM](https://github.com/user-attachments/assets/ad961446-c9ea-4168-a195-f8f2f4d60bff)